web/modules/weather_blocks/src/Plugin/Block/WeatherBlockBase.php

```diff
@@ -67,4 +67,31 @@ public function getCacheMaxAge() {
     return 0;
   }
 
+  /**
+   * Get the current location that the block should be associated with.
+   *
+   * If the current route is on the grid, pulls the WFO grid information from
+   * the URL.
+   */
```

Comment:
Leave or toss, but I was thinking some pointers in the comment might help us keep track of the structure of this location object as we modify it going forward:
```suggestion
  /**
   * Get the current location that the block should be associated with.
   *
   * If the current route is on the grid, pulls the WFO grid information from
   * the URL.
   * @returns array $location An array with location info in the following
   * structure:
   *     array['grid'] An NWS API grid response
   *.                       ['wfo'] The NWS WFO code for the location
   *                       ['gridX'] The X value of the grid coordinate
   *                       ['gridY'] The Y value of the grid coordinate
   */
```

Reply:
Good call! I was also wondering if we should create some explicit types for these sorts of things (this location, current conditions, hourly forecast, etc.). That might be overkill, though, when we can just document them at the points they're created and they're each only created in one place. 🤔